src/Microsoft.Azure.WebJobs.Host/Listeners/HostListenerFactory.cs

```diff
@@ -49,7 +47,6 @@ internal class HostListenerFactory : IListenerFactory
                 if (functionDefinition.Descriptor.IsDisabled)
                 {
                     string msg = string.Format("Function '{0}' is disabled", functionDefinition.Descriptor.ShortName);
-                    _trace.Info(msg, TraceSource.Host);
```

Comment:
Can't continue to flow category info? Or are you saying this isn't important in this case?

Reply:
Category info is already tracked in ILoggers -- when you create one, it has a category attached for every log it writes. So it's per-logger rather than per-message.